### PR TITLE
OCPBUGS-1361: Reword vSphere problem detector check alerts

### PR DESCRIPTION
**Pull request by jsafrane**
Include `oc` command to get info about detected errors. And reformatted the file a bit.

@openshift/storage

### CODE REVIEW

File: assets/vsphere_problem_detector/12_prometheusrules.yaml [hunk 1]
```diff
@@ -17,12 +17,16 @@ spec:
         labels:
           severity: warning
         annotations:
-          summary: "VSphere node health checks are failing"
+          summary: |
+            Periodic vSphere health check {{ $labels.check }} is failing on node {{ $labels.node }}.
+            To get details about the failure, please see events in namespace openshift-cluster-storage-operator:
+            'oc -n openshift-cluster-storage-operator get event --sort-by=.metadata.creationTimestamp | grep VSphere{{ $labels.check }}'
+          message: "VSphere health check {{ $labels.check }} is failing on {{ $labels.node }}."
```
Comment (bertinatto) [inline assets/vsphere_problem_detector/12_prometheusrules.yaml:24]: nit: technically the correct is `vSphere`

Comment (jsafrane) [reply assets/vsphere_problem_detector/12_prometheusrules.yaml]: Fixed

**Comment by jsafrane** [comment]
/retest
I see just flakes

**Pull request merged**